homeassistant/components/homewizard/diagnostics.py

```diff
@@ -19,6 +19,7 @@
     "unique_meter_id",
     "unique_id",
     "gas_unique_id",
+    "external_devices",
```

Comment:
TBH, this is a bit filthy. The ExternalDevice list in diagnostics return a list where the unique_id's are used as keys for the dict, we should hide that.

But.. This hides all the values in this list, which can be used for real diagnostics.

Reply:
In the meantime it seems a list is redacted recursively, which fixed this issue.